scripts/swift-version.sh

```diff
@@ -127,5 +158,5 @@ set_xcode_and_swift_versions() {
 
 return 2>/dev/null || { # only run if called directly
     set_xcode_and_swift_versions
-    echo "Found Swift version $REALM_SWIFT_VERSION in $DEVELOPER_DIR"
+    echo "Found Swift version $REALM_SWIFT_VERSION in Xcode ${REALM_XCODE_VERSION} at $DEVELOPER_DIR"
```

Comment:
Just out of curiosity, is there a reason one of the variables is expanded using `{}` and the other two aren't?

Reply:
No , I’ll fix it to be consistent